build/audit/kafka/integration-test.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# (C) Copyright IBM Corp. 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################################
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# downloads and extracts kafka to a directory
+download(){
+    if [ ! -f ${WORKSPACE}/build/audit/kafka/workarea/kafka.tgz ]
+    then 
+        curl -L -o ${WORKSPACE}/build/audit/kafka/workarea/kafka.tgz http://www-us.apache.org/dist/kafka/2.7.0/kafka_2.13-2.7.0.tgz
```

Comment:
how close does this need to be to our kafka client dependency version?
assuming they should match, we'll want to somehow remember this version is hardcoded here...

Reply:
It doesn't I left this in here... but I don't think we'll really need it - it's a easy way to flip back to downloading rather than running in docker